Wikipedia/Code/UIButton+WMFButton.m

```diff
@@ -8,30 +8,13 @@
 
 @implementation UIButton (WMFButton)
 
-+ (UIButton *)wmf_buttonType:(WMFButtonType)type handler:(void (^__nullable)(id sender))action {
++ (UIButton *)wmf_buttonType:(WMFButtonType)type target:(nullable id)target action:(nullable SEL)action {
     UIButton *button = [UIButton buttonWithType:UIButtonTypeCustom];
     button.frame = (CGRect){{0, 0}, {24, 40}};
 
     [button wmf_setButtonType:type];
 
-    [button bk_addEventHandler:^(UIButton *sender) {
-        sender.highlighted = !sender.selected; // Prevent annoying flicker.
```

Comment:
Hmm I don't recall, but could have been some odd issue with older version of iOS.